static/js/popovers.js

```diff
@@ -391,6 +402,16 @@ exports.register_click_handlers = function () {
         e.preventDefault();
     });
 
+    $('body').on('click', '.sender_info_popover .mention_user', function (e) {
+        compose.respond_to_message({trigger: 'user sidebar popover'});
+        var name = current_message_info_popover_elem.context.innerText.substr(1);
```

Comment:
I was actually suggesting we modify the popover HTML template to add data-user-id; it'd be easy to do.